test/functional/services/remote/webdriver.ts

```diff
@@ -23,10 +23,13 @@ import chromeDriver from 'chromedriver';
 // @ts-ignore types not available
 import geckoDriver from 'geckodriver';
 import { Builder, Capabilities, By, Key, logging, until } from 'selenium-webdriver';
+// import ieCapabilities from 'selenium-webdriver/lib/capabilities';
```

Comment:
Probably don't need this, but I found so many variations of docs on the IE capabilities.